homeassistant/components/xbox/base_sensor.py

```diff
@@ -62,7 +63,7 @@ def entity_picture(self) -> str | None:
         query.pop("mode", None)
         return str(url.with_query(query))
 
-    @property
+    @cached_property
```

Comment:
It looks like this can be set in the constructor in a separate PR

Reply:
#100290